components/button/index.css

```diff
@@ -0,0 +1,339 @@
+}
+
+%spectrum-BaseButton {
+  display: inline-flex;
+  box-sizing: border-box;
+
+  align-items: center;
+  justify-content: center;
+
+  /* Show the button overflow in Edge. */
+  overflow: visible;
+
+  /* Remove button the margin in Firefox and Safari. */
+  margin: 0;
+
+  border-style: solid;
+
+  /* Remove the inheritance of text transform on button in Edge, Firefox, and IE. */
+  text-transform: none;
+  -webkit-font-smoothing: antialiased;
+  /* Font smoothing for Firefox */
+  -moz-osx-font-smoothing: grayscale;
+
+  /* Correct the inability to style clickable types in iOS and Safari. */
+  -webkit-appearance: button;
+
+  /* Adjacent buttons should be aligned correctly */
+  vertical-align: top;
+
+  transition: background var(--spectrum-global-animation-duration-100) ease-out,
+              border-color var(--spectrum-global-animation-duration-100) ease-out,
+              color var(--spectrum-global-animation-duration-100) ease-out,
+              box-shadow var(--spectrum-global-animation-duration-100) ease-out;
+
+  text-decoration: none;
+  font-family: var(--spectrum-font-family-base);
+
+  line-height: var(--spectrum-button-line-height);
+
+  cursor: pointer;
+
+  &:focus {
+    outline: none;
+  }
+
+  /* Fix Firefox */
+  &::-moz-focus-inner {
+    border: 0;
+
+    /* Remove the inner border and padding for button in Firefox. */
+    border-style: none;
+    padding: 0;
+
+    /* Use uppercase PX so values don't get converted to rem */
+    margin-top: -2PX;
+    margin-bottom: -2PX;
+  }
+
+  &:disabled,
+  &.is-disabled {
+    cursor: default;
+  }
+
+  .spectrum-Icon {
+    max-height: 100%;
+    flex-shrink: 0;
+  }
+}
+
+.spectrum-Button {
+  @inherit: %spectrum-BaseButton;
+
+  border-width: var(--spectrum-button-primary-border-size);
+  border-style: solid;
+  border-radius: var(--spectrum-button-primary-border-radius);
+
+  min-height: var(--spectrum-button-primary-height);
+  height: auto;
+  min-width: var(--spectrum-button-primary-min-width);
+
+  padding: var(--spectrum-button-padding-y) calc(var(--spectrum-button-primary-padding-x) - var(--spectrum-button-primary-border-size));
+
+  /* @hack: fix button text vertical alignment at 14px */
+  padding-bottom: calc(var(--spectrum-button-padding-y) + 0.5px);
+  padding-top: calc(var(--spectrum-button-padding-y) - 0.5px);
+
+  font-size: var(--spectrum-button-primary-text-size);
+  font-weight: var(--spectrum-button-primary-text-font-weight);
+
+  &:hover,
+  &:active {
+    box-shadow: none;
+  }
+
+  .spectrum-Icon + .spectrum-Button-label {
+    margin-left: var(--spectrum-button-primary-text-gap);
+  }
+
+  .spectrum-Button-label + .spectrum-Icon {
+    margin-left: calc(var(--spectrum-button-primary-text-gap) / 2);
+  }
+}
+
+a.spectrum-Button,
+a.spectrum-ActionButton {
+  /* Remove appearance for clickable types in iOS and Safari. */
+  -webkit-appearance: none;
+  /* Make link text not selectable  */
+  user-select: none;
+}
+
+.spectrum-ActionButton {
```

Comment:
@dannify did we talk about asking for button css to be split up? or @devongovett ? can't remember

Reply:
So we can't make that change here; these are all non-breaking changes. In the future, it would be nice to split up CSS for ActionButton etc. Can you file an issue for it?